integration-cli/docker_cli_import_test.go

```diff
@@ -39,3 +42,59 @@ func (s *DockerSuite) TestImportDisplay(c *check.C) {
 	}
 
 }
+
+func (s *DockerSuite) TestImportFile(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--name", "test-import", "busybox", "true")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal("failed to create a container", out, err)
+
+	}
+
+	temporaryFile, err := ioutil.TempFile("", "exportImportTest")
+	if err != nil {
+		c.Fatal("failed to create temporary file", "", err)
+	}
+
+	runCmd = exec.Command(dockerBinary, "export", "test-import")
+	runCmd.Stdout = bufio.NewWriter(temporaryFile)
+
+	_, err = runCommand(runCmd)
```

Comment:
on all of these where you have:
```
_, err = runCommand(...)
```
you only check `err` and not the exit code, I think you should check both because `err` will just be non-nil if its not able to start the command. If the cmd itself has a non-zero exitCode 'err' will be nil.  Granted something later on in you test will probably fail but we should catch things sooner.
Just doing:
```
ec, err = ...
if ec !=0 || err != nil {
...
}
```
should cover it